docs/farming.md

```diff
@@ -188,6 +188,80 @@ After this, simply repeat the step you prompted for (step 4 or 6). This time, cl
 ```
 7. It may prompt again in here. Refer to the note on step 4.
 
+## Docker Instructions
+
+Create `subspace` directory and `docker-compose.yml` in it with following contents:
+```yml
+version: "3.7"
+services:
+  node:
+    # Replace `snapshot-DATE` with latest release
+    image: ghcr.io/subspace/node:snapshot-DATE
+    volumes:
+# Instead of specifying volume (which will store data in `/var/lib/docker`), you can
+# alternatively specify path to the directory where files will be stored, just make
+# sure everyone is allowed to write there
+      - node-data:/var/subspace:rw
+#      - /path/to/subspace-node:/var/subspace:rw
+    ports:
+# If port 30333 is already occupied by another Substrate-based node, replace all
+# occurrences of `30333` in this file with another value
+      - "0.0.0.0:30333:30333"
+# Un-comment following line to unlock node's WebSocket RPC
+#      - "127.0.0.1:9944:9944"
+    restart: unless-stopped
+    command: [
+      "--chain", "testnet",
+      "--base-path", "/var/subspace",
+      "--wasm-execution", "compiled",
+      "--execution", "wasm",
+      "--bootnodes", "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr",
+      "--port", "30333",
+      "--telemetry-url", "wss://telemetry.polkadot.io/submit/ 1",
+      "--telemetry-url", "wss://telemetry.subspace.network/submit/ 1",
+      "--rpc-cors", "all",
+      "--rpc-methods", "unsafe",
+      "--ws-external",
+      "--validator",
+# Replace `INSERT_YOUR_ID` with your node ID (shown in telemetry)
```

Comment:
also, `shown in telemetry` might be potentially confusing for some people (they may assume *oh, there should be an ID in the telemetry, I should find it and put it in here*, although they don't have a node running in the first place. I suggest rewording this as: **will be shown in telemetry**